packages/fx-core/src/plugins/resource/function/plugin.ts

```diff
@@ -636,6 +641,64 @@ export class FunctionPluginImpl {
     return ResultFactory.Success();
   }
 
+  public async generateArmTemplates(ctx: PluginContext): Promise<FxResult> {
+    const selectedPlugins = (ctx.projectSettings?.solutionSettings as AzureSolutionSettings)
+      .activeResourcePlugins;
+    const context = {
+      plugins: selectedPlugins,
+    };
+
+    const bicepTemplateDirectory = path.join(
+      getTemplatesFolder(),
+      "plugins",
+      "resource",
+      "function",
+      "bicep"
+    );
+
+    const moduleTemplateFilePath = path.join(bicepTemplateDirectory, Bicep.moduleTemplateFileName);
+
+    const parameterTemplateFilePath = path.join(
+      bicepTemplateDirectory,
+      Bicep.inputParameterOrchestrationFileName
+    );
+    const resourceTemplateFilePath = path.join(
+      bicepTemplateDirectory,
+      Bicep.moduleOrchestrationFileName
+    );
+    const outputTemplateFilePath = path.join(
+      bicepTemplateDirectory,
+      Bicep.outputOrchestrationFileName
+    );
+    const parameterFilePath = path.join(bicepTemplateDirectory, Bicep.parameterFileName);
+
+    const result: ScaffoldArmTemplateResult = {
+      Modules: {
+        functionProvision: {
+          Content: await fs.readFile(moduleTemplateFilePath, Bicep.uft8Encoding),
+        },
+      },
+      Orchestration: {
+        ParameterTemplate: {
+          Content: await fs.readFile(parameterTemplateFilePath, Bicep.uft8Encoding),
+          ParameterJson: JSON.parse(await fs.readFile(parameterFilePath, Bicep.uft8Encoding)),
```

Comment:
Please use the constant defined in packages\fx-core\src\common\constants.ts `ConstantString.UTF8Encoding`